eq-author/src/App/history/createNoteMutation.graphql

```diff
@@ -0,0 +1,15 @@
+mutation AddNote($input: CreateNoteInput!) {
+  createNote(input: $input) {
+    id
+    publishStatus
+    questionnaireTitle
+    bodyText
+    user {
+      id
+      email
+      name
+      displayName
```

Comment:
Is it enough to query only one of `name` and `displayName`? Is email needed here at all?

Reply:
Technically it is but if you don't query everything Apollo will whine as the different user objects in the cache won't quite match up.